.travis.yml

```diff
@@ -6,6 +6,8 @@ notifications:
     on_failure: change
 
 node_js:
+  - 6
+  - 5
```

Comment:
v4 will be supported for a much longer time (see e.g. the [LTS schedule](https://github.com/nodejs/LTS)) than v5, so if there’s a second additional version here, it’s probably better to choose v4 over v5.